server/src/controllers/projects.controllers.ts

```diff
@@ -253,6 +255,130 @@ class ProjectsController {
 		const keys = await projectsRepository.getKeys();
 		res.send(keys);
 	};
+
+	createLabel = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+		const projectsRepository = getCustomRepository(ProjectsRepository);
+		const { projectId, label } = req.body;
+		const { id: userId } = req.user;
+
+		try {
+			const isLabelDataValid = this.validateLabelData(label, next);
+			if (!isLabelDataValid) {
+				return;
+			}
+
+			const project = await this.getProjectById(projectId, userId, projectsRepository, next);
+
+			if (!project) {
+				return;
+			}
```

Comment:
Do we really need this?

From what I see `getProjectById` itself will throw an error if it cannot find project
```
		if (project === undefined) {
			next(new ErrorResponse(HttpStatusCode.NOT_FOUND, projectsErrorMessages.PROJECT_NOT_FOUND));
			return null;
		}
```

If we can remove it here then it should be removed from `updateLabel` and `deleteLabel` too.